### PR TITLE
fix permission loop in ocr scanner

### LINKED ISSUE #31: Stuck in a loop on deny permissions
When you deny permissions in the scan passport activity you can't escape. Pressing ok on either of the permission explanation messages should close the activity.

### PR DESCRIPTION
**Pull request by wkmeijer**
When permissions are denied, the manual input activity is opened.
Fixes #31

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -28,7 +28,7 @@
     <string name="create_key">Create new key</string>
     <string name="show_key">Show key</string>
     <string name="verify_sig">Verify signature</string>
-    <string name="storage_permission_explanation">Give permission for storage!</string>
+    <string name="storage_permission_explanation">Please give storage permissions for scanning the text on the passport</string>
```
Comment (jonathanraes) [inline app/src/main/res/values/strings.xml]: I believe this explanation is re-used in some places

**Comment by klikooo** [comment]
I got an exception when I accept the permissions and then go back on install:
```
    java.util.NoSuchElementException
        at java.util.ArrayList$Itr.next(ArrayList.java:834)
        at java.util.Collections.max(Collections.java:742)
        at nl.tudelft.cs4160.trustchain_android.passport.ocr.TesseractOCR.giveStats(TesseractOCR.java:227)
        at nl.tudelft.cs4160.trustchain_android.passport.ocr.TesseractOCR.cleanup(TesseractOCR.java:206)
        at nl.tudelft.cs4160.trustchain_android.passport.ocr.TesseractOCR$3.run(TesseractOCR.java:124)
        at android.os.Handler.handleCallback(Handler.java:751)
        at android.os.Handler.dispatchMessage(Handler.java:95)
        at android.os.Looper.loop(Looper.java:154)
        at android.os.HandlerThread.run(HandlerThread.java:61)
```
Quick fix might be to remove the giveStats funtion

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (wkmeijer) [reply app/src/main/res/values/strings.xml]: Ok, I created a separate one for the ocr.

**Comment by jonathanraes** [comment]
@klikooo very strange crash i got the same

**Comment by jonathanraes** [comment]
I suppose it happens because the times list is still empty, could have added an if but its not _that_ important

**Comment by wkmeijer** [comment]
Ok, I'll do that, that's ab it less lazy.

**Comment by klikooo** [comment]
It works now!

**Pull request merged**